tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -33,7 +33,10 @@ import (
 // https://sdk.operatorframework.io/docs/best-practices/observability-best-practices/#metrics-guidelines
 // should be ignored.
 var excludedMetrics = map[string]struct{}{
-	"kubevirt_vmi_phase_count": struct{}{},
+	"kubevirt_vmi_phase_count":                  {},
```

Comment:
Why do you want to ignore them?

Reply:
this collector is used by the linter, and these metrics do not follow the naming conventions, that's why I'm excluding them from the linter, while we don't refactor them

Reply:
maybe worth changing the names to start with kubevirt_?

Reply:
I agree we need to do so, but I don't think it should be as part of this PR

Reply:
@machadovilaca  Please add `kubevirt_` prefix to all the metrics.
You already added `reflector_` to the reflector metrics.

Reply:
I'm avoiding renaming any metrics as part of this PRs
reflector was already part of the metrics names through the 'subsytem': https://github.com/kubevirt/kubevirt/blob/055c6e0491fa93befa6372ca4d367916cabcb5af/pkg/monitoring/reflector/prometheus/prometheus.go#L26

I'm also avoiding using the subsystem method because of this, IMO it makes it harder to find metrics by name and it is easier to generate confusion